.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "esbonio.sphinx.confDir": ""
+}
```

Comment:
If we're going to commit changes for vscode, let's do it in a separate PR. If you didn't intend to commit this file, [you can add it to your `.git/info/exclude` file which is the same as .gitignore, but just for your machine](https://stackoverflow.com/a/1753078/1637351).

Reply:
I did not mean to commit this, and I forgot to remove the comments, I will rectify in the next update.